lib/mayaUsd/render/mayaToHydra/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ target_sources(${TARGET_NAME}
 )
 
 # -----------------------------------------------------------------------------
-# compile definitions
+# compiler configuration
 # -----------------------------------------------------------------------------
 target_compile_definitions(${TARGET_NAME}
     PRIVATE 
```

Comment:
I don't believe I have `$<$<BOOL:${IS_LINUX}>:LINUX>` any where in `mayaUsd_compile_config`?

 I also wouldn't add this to the mayaUsd_compile_config since not all targets really need this definition.

Reply:
Ahh, sorry, my mistake. Apparently I didn't scroll up far enough. :)

I saw this one:
https://github.com/Autodesk/maya-usd/blob/20ea6b767e710455dc30eeb65b3d89301378766d/lib/mayaUsd/CMakeLists.txt#L51

But failed to notice that was in mayaUsd's CMakeLists.txt, not `mayaUsd_compile_config()`. Ignore me!